lms/djangoapps/certificates/models.py

```diff
@@ -436,6 +436,22 @@ def get_certificate_invalidations(cls, course_key, student=None):
             })
         return data
 
+    @classmethod
+    def is_certificate_invalidated(cls, course_key, student):
+        """Check that certificate for a student is invalidated or not.
+
+        Arguments:
+            course_key (CourseKey): The course to associated with certificate.
```

Comment:
"The course 'to be' associated with the certificate" or "The course associated with the certificate"